build-scripts/compile_product.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+
+import ssg.products
+
+
```

Comment:
What are the pros and  cons of having this a separate script versus including this to compile_all.py?

Reply:
I think that before the functionality is stable, having it as a fast and separate step is convenient for playing around with. I don't think that the separate step causes any problems or notable inefficiencies.